modules/pel/peltool/peltool.py

```diff
@@ -372,6 +389,8 @@ def main():
                         help='Delete original file after parsing')
     parser.add_argument('-t', '--termination', dest='critSysTerm',
                         action='store_true', help='List only critical system terminating PELs')
+    parser.add_argument('-a', '--all-pels', dest='all',
```

Comment:
if we're going to go the direction of the '-l' with the '-a' for nonBMC, the argument options would need to spell that out or people will have no idea how to make it work.

so if not inBMC, the -a description would need to say it requires the -l, and then also code would have to enforce it.

Reply:
How about if we use -a option as we do -l?
In non-BMC system, we can pass path to folder directly with -a option? Cause as of now -a is a standalone functionality.